src/generic/grideditors.cpp

```diff
@@ -1679,6 +1676,19 @@ wxString wxGridCellChoiceEditor::GetValue() const
   return Combo()->GetValue();
 }
 
+void wxGridCellChoiceEditor::onComboCloseUp(wxCommandEvent& WXUNUSED(evt))
+{
+    wxGridCellEditorEvtHandler* evtHandler = wxDynamicCast(m_control->GetEventHandler(),
```

Comment:
Another thing: could we avoid this repeated call to dynamic cast if we just caught `CLOSEUP` event in `evtHandler` itself? It should live for long enough, AFAICS, and this could make things simpler...

Reply:
I did it this way to avoid giving the event handler knowledge about the editors themselves. Also, I think it is more visible if it is associated with the editor: this way looking at the editor code it is obvious the combobox event is being handled.

Reply:
This is a good point, thanks.